Libplanet.Tests/Store/BlockSetTest.cs

```diff
@@ -92,7 +92,12 @@ public void CanRemoveItem()
             Assert.False(_set.Remove(_fx.Block1.Hash));
             _set[_fx.Block1.Hash] = _fx.Block1;
             Assert.True(_set.Remove(_fx.Block1.Hash));
-            Assert.Throws<KeyNotFoundException>(() => { var val = _set[_fx.Block1.Hash]; });
+            Assert.Throws<KeyNotFoundException>(() =>
+            {
+#pragma warning disable SA1312
+                var _ = _set[_fx.Block1.Hash];
+#pragma warning restore SA1312
+            });
```

Comment:
```suggestion
            Assert.Throws<KeyNotFoundException>(() => _ = _set[_fx.Block1.Hash]);
```

I think this would work.

Also a small comment like 

```csharp
#pragma warning disable SA1312 // Variable '_' should begin with lower-case letter
```

would clarify to a later maintainer as to why such pragma warning is disabled (although this becomes irrelevant if we assign to `_` without `var` 🙃).